src/components/LeftSidebar/ConversationsList/AppContentListItem/AppContentListItem.vue

```diff
@@ -25,42 +25,57 @@
 		v-bind="navElement">
 		<a
 			:id="anchorId"
+			ref="acli"
```

Comment:
holy, this should all be upstreamed... when is this planned?

Reply:
After the messagelist virtual scroller